src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -87,3 +87,21 @@
         {{ label|trans({}, 'Admin.Advparameters.Feature') }}
     </a>
 {% endmacro %}
+
+{# Show form widget with errors rendered below it #}
+{% macro form_widget_with_error(form, vars) %}
```

Comment:
use native form_errors?

Reply:
if i used it, it works, but it looks ugly. :/

Reply:
We maybe need to improve the default style, it will be easier to everyone to use what twig & symfony allow us to used :)

Reply:
thats an option. :)